test/e2e/specs/editor/plugins/inner-blocks-prioritized-inserter-blocks.spec.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Prioritized Inserter Blocks Setting on InnerBlocks', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-innerblocks-prioritized-inserter-blocks'
+		);
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-innerblocks-prioritized-inserter-blocks'
+		);
+	} );
+
+	test.describe( 'Quick Inserter', () => {
+		test( 'uses defaulting ordering if prioritzed blocks setting was not set', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( {
+				name: 'test/prioritized-inserter-blocks-unset',
+			} );
+
+			const block = page.getByRole( 'document', {
+				name: 'Block: Prioritized Inserter Blocks Unset',
+			} );
+			await expect( block ).toBeVisible();
```

Comment:
nit: Generally, this should be unnecessary since the [actionability checks for click](https://playwright.dev/docs/actionability#:~:text=%2D-,click,%2D,-dblclick) (below) include the element visibility.